python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -1019,24 +1019,6 @@ $defs:
             min: 0
             max: 300
             default: 30
-          inband_management_subnet:
-            documentation_options:
-              table: node-type-inband-management-configuration
-            deprecation:
-              warning: true
-              new_key: inband_mgmt_subnet
-              remove_in_version: 5.0.0
-            type: str
-            $ref: "eos_designs#/$defs/node_type/keys/defaults/keys/inband_mgmt_subnet"
-          inband_management_vlan:
-            documentation_options:
-              table: node-type-inband-management-configuration
```

Comment:
This needs to be included in the removed keys as well, Otherwise it will not show up in the correct documentation table. Thanks.

Reply:
updated